common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -404,6 +404,35 @@ def test_video_with_youtube_blocked(self):
 
         self.assertTrue(self.video.is_video_rendered('html5'))
 
+    def test_html5_video_caption_fetch_with_youtube_id(self):
```

Comment:
Maybe this makes more sense --> `test_html5_video_rendered_with_youtube_captions`